README.md

```diff
@@ -73,9 +73,11 @@ cross-site data in these reports would be encrypted to ensure it can only be
 processed by the aggregation service. During processing, this service will add
 noise and impose limits on how many queries can be performed.
 
-This API introduces a `sendHistogramReport()` function; see
-[examples](#examples) below. This call constructs an aggregatable report, which
-contains an encrypted payload for later computation via the aggregation service.
+This API introduces a `contributeToHistogram()` function; see
+[examples](#examples) below. This call saves a histogram contribution for
```

Comment:
Wdyt if we say "This call registers a histogram contribution for reporting."

Reply:
That does sound better! Done